src/main/interface/settings.c

```diff
@@ -257,12 +257,28 @@ static const char * const lookupTableLowpassType[] = {
     "BIQUAD",
 #if defined(USE_FIR_FILTER_DENOISE)
     "FIR",
+#else
+    NULL,
 #endif
     "BUTTERWORTH",
     "BIQUAD_RC_FIR2",
     "FAST_KALMAN"
 };
 
+static const char * const lookupTableDtermLowpassType[] = {
+    "PT1",
+    "BIQUAD",
+#if defined(USE_FIR_FILTER_DENOISE)
+    "FIR",
+#else
+    NULL,
+#endif
+    NULL,
```

Comment:
Not sure if we need the trailing `NULL` here, and I suspect the array entries will not be optimised away.

Reply:
Looking at the code, trailing NULLs are not necessary.